pkg/webui/components/field/index.js

```diff
@@ -215,6 +217,8 @@ const Field = function (props) {
       <Component
         className={style.component}
         id={id}
+        component={type === 'textarea' ? 'textarea' : undefined}
```

Comment:
imo it would be better to make a standalone `<TextArea />` component and clutter the `<Input />` props

Reply:
I disagree. This is basically one new line vs. a whole new component. The only difference being what HTML element is rendered. [React itself already helps](https://reactjs.org/docs/forms.html#the-textarea-tag) streamlining the api of `textarea` and `input`.

Reply:
1. `<input /> !== <textarea />`. it has other attributes that are not documented.
2. Most popular UI component libraries treat `textarea` as a separate component, [semantic-ui](https://react.semantic-ui.com/collections/form/#shorthand-subcomponent-control), [ant-design](https://github.com/ant-design/ant-design/blob/master/components/input/TextArea.tsx)
3. We already have logic in `<Field />` to switch on field types.
4. `component={type === 'textarea' ? 'textarea' : undefined}` doesnt look right

Reply:
I see your point. What I am concerned about is the repetition, as the two components would be 95% equal. So we would best do some sort of composition to avoid redundant code. I don't feel like doing that now as we have more urgent things to do and the current implementation works sufficiently. I'll make sure to open an issue about it, would you be ok with that?

Additionally, I have removed the `component={type === 'textarea' ? 'textarea' : undefined}` and solve this properly inside the input component, as it should be.